internal/command/migrate_to_v2/volumes.go

```diff
@@ -53,7 +56,6 @@ func (m *v2PlatformMigrator) validateVolumes(ctx context.Context) error {
 }
 
 func (m *v2PlatformMigrator) migrateAppVolumes(ctx context.Context) error {
-
 	m.appConfig.SetMounts(lo.Map(m.appConfig.Mounts, func(v appconfig.Mount, _ int) appconfig.Mount {
 		v.Source = nomadVolNameToV2VolName(v.Source)
 		return v
```

Comment:
Personally, I'd prefer it ends up in here, both because this is where the other patch happens to migrate the volume config, and because modifying in a validation function is easy to miss.
```suggestion
	m.appConfig.SetMounts(lo.Map(m.appConfig.Mounts, func(v appconfig.Mount, _ int) appconfig.Mount {
	    if len(v.Processes) == 0 {
	        v.Processes = m.appConfig.ProcessNames()
	    }
		v.Source = nomadVolNameToV2VolName(v.Source)
		return v
```

Reply:
totally makes sense. thanks